ADR/0004-component-image-location.md

```diff
@@ -0,0 +1,190 @@
+# 4. Out-of-the-box image repository for StoneSoup users 
+
+Date: Oct 29, 2022 
+
+## Status
+
+Accepted
+
+## Approvers
+
+* Alexey Kazakov
+* Gorkem Ercan
+
+## Reviewers
+
+* Alexey Kazakov
+* Gorkem Ercan
+* Parag Dave
+* Ralph Bean
+* Andrew McNamara
+
+
+## Context
+
+**Problem**
+StoneSoup does not have a internal registry where images could be pushed to as an intermediate step before being deployed as a container.
+As an application centric experience, StoneSoup should not make it mandatory for its users to specify where the image ( built from source code ) needs
+to be pushed to.
+
+
+**Goals**: 
+* Provide an out-of-the-box location for users’ images to be pushed after being built from source.
+
+
+**Non-Goals**: 
+* Define the user experience for users bringing existing images from other image registry services.
+* Provide users an option to choose what the out-of-the-box location for images would be. 
+* Define the user experience for users who are willing to configure additional credentials for pushing to an image registry of their choice.
+
+**Design Goals**
+* Use Quay.io as the out-of-the-box image registry in order to avoid getting into the business of maintaining an internal registry.
+* Align user permissions in an StoneSoup workspace with those in Quay.io
+* Maintain the right levels of isolation between images being expected of a multi-tenant system.
```

Comment:
What are the levels of isolation? Is isolation level between images explained anywhere else in any AppStudio document?